### DISCUSSION TITLE
disable mousescroll in cssmode

### DISCUSSION THREAD

**Discussion by thebestgin** [nolimits4web/swiper#4281] [Q&A]
I would like to use cssmode. 
But at the same time I want to turn off mousescroll.
Unfortunately the swiper api does not work for this.

What are the possibilities?
Does swiper have a bug in cssmode?

**Comment by nolimits4web** [comment]
```js
document.querySelector('.swiper-container').addEventListener('wheel', (e) => e.preventDefault(), true)
```
